hadoop-master1/files/etc/trino/conf/trino-env.sh

```diff
@@ -0,0 +1 @@
+export JAVA_HOME=/opt/openjdk-21
```

Comment:
emm... this file seems does not take effect

Reply:
Yes, old test file, will remove it.